lib/matplotlib/axes/_axes.py

```diff
@@ -2705,11 +2705,11 @@ def errorbar(self, x, y, yerr=None, xerr=None,
 
         Parameters
         ----------
-        x : scalar
-        y : scalar
+        x : scalar or array-like
```

Comment:
I think that's good enough.

FWIW git-grepping the codebase of scipy and skimage yields *no* result for `... or array-like` or `array-like or ...` (where ... refers to "scalar", "float", or some similar term); meanwhile at least some functions that are documented to take an "array-like" as argument can also take a scalar.

scikit-learn is different because it documents the shape of the required input nearly every time it documents an input as array-like, so scalars are explicitly excluded by that shape info.

sympy uses the word array-like exactly twice in its docs (in the sense of not including scalars).

pandas seems to use the word array-like in the sense of not including scalars as well.

So overall I wouldn't say there's any global agreement.